CMake/PackageDebian.cmake

```diff
@@ -37,5 +37,3 @@ if (UNIX)
         "${CPACK_DEBIAN_PACKAGE_NAME}_${CPACK_DEBIAN_PACKAGE_VERSION}_${CPACK_DEBIAN_PACKAGE_ARCHITECTURE}")
 
 endif(UNIX)
-
-INCLUDE(CPack)
```

Comment:
What are the consequences of this?

Reply:
I don't believe there are any negative consequences. I have a windows packager configuration that needs to be polished up for a merge request after I have fed back all the changes from this request.